config/postfix-accounts.cf

```diff
@@ -0,0 +1 @@
+# TODO is this file even used?
```

Comment:
I think this and the other similar files are placeholders that make it easy for new user to find where the can make changes. The .bak file has probably been checked in by mistake though.

Reply:
It feel a bit like this is a relic of a change of the past. I did not immediately find where this file is put into the container and therefore decided to add this todo.